src/app/shared/request/request.interceptor.ts

```diff
@@ -34,7 +34,10 @@ export class RequestInterceptor implements HttpInterceptor {
     }
 
     // do not need to pass team id for teams.json
-    if (teamId && !req.url.includes('/teams.json')) {
+    // do not need to pass team id for chat api calls
+    if (teamId && !req.url.includes('/teams.json') && 
+    !req.url.includes('/message/chat/list.json') && !req.url.includes('/message/chat/create_message') && 
```

Comment:
this can be done when user in one team but there is a way that user in multiple teams in that case i need to pass team ID to api in body. that's why i pass team ID in body.